common/src/stack/command/stack/commands/config/host/interface/__init__.py

```diff
@@ -146,5 +147,5 @@ def run(self, params, args):
 
 		post_config = self.command('list.host.interface', [host])
 
-		if pre_config != post_config:
+		if pre_config != post_config and not stack.bool.str2bool(no_sync_config):
```

Comment:
If you change the parameter to `sync` and convert it above, then this line becomes:
```
if pre_config != post_config and sync:
```